apps/patient/src/utils/general.ts

```diff
@@ -104,11 +106,22 @@ function isCloseEvent(event: types.PharmacyEvent): event is types.PharmacyCloseE
   return event.type === 'close';
 }
 
-export const preparePharmacyHours = (pharmacy: types.Pharmacy): EnrichedPharmacy => {
+export const preparePharmacy = (pharmacy: types.Pharmacy): EnrichedPharmacy => {
   let is24Hr = false;
   let isClosingSoon = false;
   let opens = '';
   let closes = '';
+  let isUrgent = false;
```

Comment:
What does `isUrgent` mean for a pharmacy?

Reply:
that surface the ready in 30min badge

Reply:
why not `showReadyIn30min`?